crypto/bio/bss_acpt.c

```diff
@@ -150,7 +150,7 @@ static int acpt_free(BIO *a)
 static int acpt_state(BIO *b, BIO_ACCEPT *c)
 {
     BIO *bio = NULL, *dbio;
-    int s = -1, ret = -1;
+    int s = -1, ret = -1, sock = -1;
```

Comment:
Why add another variable for the socket when you have `s`?

Reply:
I think you're right. Amended.